src/components/Picker/Picker.tsx

```diff
@@ -34,20 +33,17 @@ export interface Props {
   chipComponent?: React.ReactNode;
   searchResultComponent?: React.ReactNode;
   moreInfoComponent?: React.ReactNode;
-  source: IPickerSource<IPickerInfo>;
   moreInfoComponentShowOn?: DisplayMoreInfo;
   style?: React.CSSProperties;
   theme?: any;
   autoSuggest?: boolean;
   autoSuggestMethods?: object[];
-  itemsList?: object[];
   stateProps?: {chipListState: any, suggestions: any, inputProps: object, value?: any};
   searchBehavior?(): void;
   onSelect?(item: any): void;
   onRemove?(item: any): void;
```

Comment:
For item everywhere, instead of `any` for item type, we should look into using generics.  But, this is more a nice to have, if its too much work, its not worth it.